ui/src/DialogResponses.ts

```diff
@@ -7,8 +7,14 @@ import { MessageItem } from 'vscode';
 import { localize } from './localize';
 
 export namespace DialogResponses {
+    export const yes: MessageItem = { title: localize('yes', 'Yes') };
+    export const no: MessageItem = { title: localize('no', 'No') };
+    export const cancel: MessageItem = { title: localize('cancel', 'Cancel'), isCloseAffordance: true };
+    export const deleteResponse: MessageItem = { title: localize('delete', 'Delete') };
+    export const learnMore: MessageItem = { title: localize('learnMore', 'Learn more') };
+    export const dontWarnAgain: MessageItem = { title: localize('dontWarnAgain', 'Don\'t warn again') };
+    export const skipForNow: MessageItem = { title: localize('skipForNow', 'Skip for now') };
     export const upload: MessageItem = { title: localize('upload', "Upload") };
-    export const dontWarn: MessageItem = { title: localize('dontwarn', "Upload, don't warn again") };
+    export const uploadDontWarnAgain: MessageItem = { title: localize('dontwarn', "Upload, don't warn again") };

```

Comment:
Can we change to "Always upload"?

Reply:
I don't have a preference - but if someone else complains about the change I'm pointing them directly to you :P